### PR TITLE
Update references and documentation

### PR DESCRIPTION
**Pull request by cduverne**
This PR updates the documentation of the repo

### CODE REVIEW

**Comment by cduverne** [comment]
cc @fdemesmaeker

File: README.md [hunk 1]
```diff
@@ -48,24 +48,24 @@ Please file a issue request on the [issues](https://github.com/euranova/docker-m
 
 # Installation
 
-Automated builds of the image are available on [Dockerhub](https://hub.docker.com/r/jasl8r/mattermost) and is the recommended method of installation.
+Automated builds of the image are available on [Dockerhub](https://hub.docker.com/r/euranova/docker-mattermost) and is the recommended method of installation.
 
-> **Note**: Builds are also available on [Quay.io](https://quay.io/repository/jasl8r/mattermost)
+> **Note**: Builds are also available on [Docker Hub](https://hub.docker.com/r/euranova/docker-mattermost)
```
Comment (fdemesmaeker) [inline README.md:53]: This note looks like a duplicate of the previous line

File: README.md [hunk 2]
```diff
@@ -48,24 +48,24 @@ Please file a issue request on the [issues](https://github.com/euranova/docker-m
...
 ```bash
-docker pull jasl8r/mattermost:3.5.1
+docker pull euranova/docker-mattermost:latest
 ```
 
 You can also pull the `latest` tag which is built from the repository *HEAD*
 
 ```bash
-docker pull jasl8r/mattermost:latest
+docker pull euranova/docker-mattermost:latest
```
Comment (fdemesmaeker) [inline README.md:62]: Either remove this section about "pull the latest tag" or add a version number at line 56

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,9 +4,14 @@ This file only reflects the changes that are made in this image. Please refer to
...
 for the list of changes in Mattermost.
 
-## 3.5.1
+## 4.9.0
 
-- mattermost 3.5.1
+- Mattermost 4.9.0
+- Refactoring of the image 
+
+## 4.9.0
```
Comment (fdemesmaeker) [inline CHANGELOG.md]: Two times the 4.9.0 section?

**Review by fdemesmaeker — COMMENTED** [review]
LGTM except these 3 comments

File: README.md (hunk 1 above)
Comment (cduverne) [reply README.md:53]: Yes but no, look at the syntax

File: CHANGELOG.md (hunk 3 above)
Comment (cduverne) [reply CHANGELOG.md]: Reviewed

**Pull request merged**